### PR TITLE
Start a workload.

### PR DESCRIPTION
**Pull request by tbl3rd**
https://broadinstitute.atlassian.net/browse/GH-684

### CODE REVIEW

File: src/zero/service/postgres.clj [hunk 1]
```diff
@@ -93,6 +95,61 @@
                          (util/getenv "USER" "postgres")
                          "password")))
 
+(defn get-table
+  "Return TABLE using transaction TX."
+  [tx table]
+  (jdbc/query tx (format "SELECT * FROM %s" table)))
```
Comment (rexwangcc) [inline src/zero/service/postgres.clj:101]: Does jdbc provides built-in protection against SQL injections? Cannot find any useful info https://github.com/clojure/java.jdbc

**Comment by tbl3rd** [comment]
Regarding: `[:response {:message "Unauthorized"}]`.
You can get that if you are not authenticated to gcloud or vault.
Otherwise, it should succeed against a local `./ops/server.sh` server.
(Assuming someone has run `:test-create-workload` first so there is a workload to start.

File: integration/zero/start.clj [hunk 2]
```diff
@@ -0,0 +1,49 @@
+(ns zero.start
+  (:require [clojure.data.json :as json]
+            [clojure.pprint :refer [pprint]]
+            [zero.util :as util])
+  (:import (java.util UUID)))
+
+(def server
+  "https://wfl-dot-broad-gotc-dev.appspot.com"
```
[the remark this answers is not in the document]
Comment (tbl3rd) [reply integration/zero/start.clj:8]: Yes. Right ow you just have to hack that to point where you want.
We could switch on a command-line argument or something.

File: src/zero/api/handlers.clj [hunk 3]
```diff
@@ -113,25 +115,56 @@
...
+      (->> (if-let [uuid (get-in request [:parameters :query :uuid])]
+             [{:uuid uuid}]
+             (jdbc/query tx ["SELECT uuid FROM workload"]))
+           (mapv (partial postgres/get-workload-for-uuid tx))
+           succeed))))
+
+(defn post-start
+  "Start the workloads with UUIDs in REQUEST."
+  [request]
+  (let [start {"AllOfUsArrays"                   aou/start-workload!
```
[the remark this answers is not in the document]
Comment (tbl3rd) [reply src/zero/api/handlers.clj:154]: I agree. Who cares?

File: src/zero/module/wgs.clj [hunk 4]
```diff
@@ -134,7 +133,7 @@
                           hack-task-level-values))
         {:keys [destination_cloud_path final_gvcf_base_name]} inputs
         output (str destination_cloud_path final_gvcf_base_name ".cram")]
-    (all/throw-when-output-exists-already! output)
+    #_(all/throw-when-output-exists-already! output)
```
[the remark this answers is not in the document]
Comment (tbl3rd) [reply src/zero/module/wgs.clj:136]: Good catch. I should back that out.

File: src/zero/module/wl.clj [hunk 5]
```diff
@@ -0,0 +1,105 @@
+(ns zero.module.wl
```
[the remark this answers is not in the document]
Comment (tbl3rd) [reply src/zero/module/wl.clj:1]: We'll figure that out over time, but I hope not.

File: src/zero/service/postgres.clj (hunk 1 above)
Comment (tbl3rd) [reply src/zero/service/postgres.clj:101]: I don't see a path to injection, but we could use prepared statements.

**Pull request merged**